src/db.rs

```diff
@@ -192,25 +186,25 @@ pub async fn get_app_info_for_app_id(
     app_id: &str,
 ) -> sqlx::Result<AppInfo> {
     sqlx::query_as::<_, DBAppInfo>(
-        "SELECT * FROM app_infos WHERE app_id = ? ORDER BY version DESC LIMIT 1;",
+        "SELECT * FROM app_infos WHERE app_id = ? ORDER BY tag_name DESC LIMIT 1;",
     )
     .bind(app_id)
     .fetch_one(c)
     .await
     .map(|app| app.into())
 }
 
-/// Get app_info with greate version.
-pub async fn maybe_get_greater_version(
+/// Returns wheter an app_info with greater tag_name exists.
```

Comment:
Isn't it sufficient to just check "old != new" and to not demand tag-names to be ordered? 
Even if there is a downgrade (maybe a tag/release got blacklisted by the store) it should still change on the frontend.

Reply:
Beforehand I implemented the version as a serial kind of thing. So it would only increase but never decrease - with tag_name that is no longer the case. The underlying code still makes the initial assumption at some code parts so I need some deeper analysis to make this generalization work for sure. I have that on my plan, but I want to make a follow-up PR for that.